src/stores/game.js

```diff
@@ -436,6 +447,7 @@ export const useGameStore = defineStore('game', {
         io.socket.get(`/api/game/${gameId}?gameStateIndex=${gameStateIndex}`, (res, jwres) => {
           switch (jwres.statusCode) {
             case 200:
+              this.resetState();
```

Comment:
resetting state clears out dialogs and overlays. Need to be careful to ensure id, and myPNum always get set